ssv/spectest/tests/valcheck/valcheckattestations/attestation_data_nil.go

```diff
@@ -20,7 +20,7 @@ func ConsensusDataNil() tests.SpecTest {
 	return &valcheck.SpecTest{
 		Name:          "consensus data value check nil",
 		Network:       types.PraterNetwork,
-		BeaconRole:    types.BNRoleAttester,
+		RunnerRole:    types.RoleCommittee,
 		Input:         input,
```

Comment:
Do you mean a BeaconVote field instead of `Input`? I think we use a []byte because the qbft module uses it and it can either be a ConsensusData or BeaconVote depending on the RunnerRole

Reply:
yeah but current test encodes bytes with consensusData